connect-api/publish.gradle

```diff
@@ -64,42 +51,16 @@ install {
     }
 }
 
-version = libraryVersion
+version = libVersion
 
-task sourcesJar(type: Jar) {
-    archiveClassifier.set("sources")
-    from android.sourceSets.main.java.srcDirs
-}
-
-task javadoc(type: Javadoc) {
-    source = android.sourceSets.main.java.srcDirs
-    classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-}
-
-afterEvaluate {
-    javadoc.classpath +=
-            files(android.libraryVariants.collect { variant -> variant.javaCompileProvider.get().classpath.files
-            })
-}
-
-task javadocJar(type: Jar, dependsOn: javadoc) {
-    archiveClassifier.set("javadoc")
-    from javadoc.destinationDir
-}
-
-artifacts {
-    archives javadocJar
-    archives sourcesJar
-}
+bintray {
 
-if (project.hasProperty('bintrayUser') && project.hasProperty('bintrayKey') &&
-        project.hasProperty('bintrayGpgPassphrase')) {
-    bintray.user = project.bintrayUser
-    bintray.key = project.bintrayKey
-    bintray.pkg.version.gpg.passphrase = project.bintrayGpgPassphrase
-}
+    Properties properties = new Properties()
+    properties.load(project.rootProject.file('local.properties').newDataInputStream())
+    user = properties.getProperty('bintra1y.user')
```

Comment:
ah typo.

Reply:
do we need `bintray1`? Shouldn't it just be `bintray`?

Reply:
Ah no, typo. We should use without the 1.